docs/semgrep-app/managing-policy.md

```diff
@@ -7,6 +7,21 @@ import MoreHelp from "/src/components/MoreHelp"
 
 # Managing CI policy
 
+:::info
+
+  Policy management is an advanced feature
+  for orchestrating complex
+  rule and notification channel assignments.
+  
+  | **Semgrep App Tier** | **Signup date** | **Availability** |
+  |----------|-----------------|------------------|
+  | Enterprise     | Any             | available        |
+  | Team     | Any             | available        |
+  | Community     | before September 15, 2021             | available        |
+  | Community     | after September 15, 2021             | unavailable, use the [Rule board](rule-board.md) instead.|
```

Comment:
Could the table be replaced by one or two sentences?

For example:

This feature is available to all users on the Enterprise and Team tiers, and Community tier users who signed up before September 15, 2021. Community tier users who signed up after September 15, 2021 should use the Rule board instead.

I _think_ that’s a bit easier to parse.

Reply:
I agree 😄

Reply:
heey this is what I was asking you about last Friday :D

Reply:
👀